### PR TITLE
closes #1752 replace admiral.test in examples@devel

### LINKED ISSUE #1752: General Issue: Replacing admiral.test calls in the function examples
### Background Information

There are several functions that using `admiral.test` data in the function examples, these data calls need to be replace with data created calling `tribble()`. A list of the files with the `library(admiral.test)` call is below. Please note the list below is a living list since some functions are being deprecated and `library(admiral.test)` might have already being removed (see issues #1681, #1682 #1683  #1723 for the functions/files where the example section will be removed for the relevant files from the list below and issue #1751 for discussion surrounding this current issue )

image

### Definition of Done

- [ ] Prioritize fixing this - https://github.com/pharmaverse/admiral/issues/1780
- [ ] All `library(admiral.test)`  have been removed from the example sections for the aforementioned files
- [ ] Example data in the example sections are created by calling  `tribble()`
- [ ] NEWS.md  is updated

### PR DESCRIPTION
**Pull request by StefanThoma**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by galachad** [comment]
Hi @StefanThoma, I changed the target branch to devel.

**Comment by StefanThoma** [comment]
@pharmaverse/admiral 
There are two issues: 


1. The styler package (in my opinion) does a bad job with tribbles:
Screenshot 2023-05-22 at 13 19 15

2. in `derive_vars_last_dose.R` I had to deal with a really wide tribble. I did this by `cbind()`-ing two more narrow tribbles: 

```
 ex_single <- cbind(
   tribble(
     ~STUDYID, ~DOMAIN, ~USUBJID, ~EXSEQ, ~VISIT,
     "PILOT1", "EX", "18-1066", 1L, "BASE",
     "PILOT1", "EX", "18-1066", 2L, "BASE",
     "PILOT1", "EX", "18-1066", 3L, "BASE",
     "PILOT1", "EX", "18-1066", 4L, "BASE",
     "PILOT1", "EX", "18-1066", 5L, "BASE",
     "PILOT1", "EX", "18-1066", 6L, "BASE",
     "PILOT1", "EX", "18-1066", 7L, "BASE",
     "PILOT1", "EX", "18-1066", 8L, "BASE",
     "PILOT1", "EX", "18-1066", 9L, "BASE",
     "PILOT1", "EX", "18-1066", 10L, "BASE",
     "PILOT1", "EX", "10-1083", 1L, "BASE",
     "PILOT1", "EX", "10-1083", 2L, "BASE",
     "PILOT1", "EX", "10-1083", 3L, "BASE",
     "PILOT1", "EX", "10-1083", 4L, "BASE",
     "PILOT1", "EX", "10-1083", 5L, "BASE",
     "PILOT1", "EX", "10-1083", 6L, "BASE",
     "PILOT1", "EX", "10-1083", 7L, "BASE",
     "PILOT1", "EX", "10-1083", 8L, "BASE",
     "PILOT1", "EX", "10-1083", 9L, "BASE",
     "PILOT1", "EX", "10-1083", 10L, "BASE",
     "PILOT1", "EX", "10-1083", 11L, "BASE"
   ),
   tribble(
     ~EXDOSE, ~EXTRT, ~EXENDTC, ~EXSTDTC, ~EXDOSFRQ,
     54, "XANO", "2013-07-07", "2013-07-07", "ONCE",
     54, "XANO", "2013-07-08", "2013-07-08", "ONCE",
     54, "XANO", "2013-07-09", "2013-07-09", "ONCE",
     54, "XANO", "2013-07-10", "2013-07-10", "ONCE",
     54, "XANO", "2013-07-11", "2013-07-11", "ONCE",
     54, "XANO", "2013-07-12", "2013-07-12", "ONCE",
     54, "XANO", "2013-07-13", "2013-07-13", "ONCE",
     54, "XANO", "2013-07-14", "2013-07-14", "ONCE",
     54, "XANO", "2013-07-15", "2013-07-15", "ONCE",
     54, "XANO", "2013-07-16", "2013-07-16", "ONCE",
     0, "PLACEBO", "2013-07-22", "2013-07-22", "ONCE",
     0, "PLACEBO", "2013-07-23", "2013-07-23", "ONCE",
     0, "PLACEBO", "2013-07-24", "2013-07-24", "ONCE",
     0, "PLACEBO", "2013-07-25", "2013-07-25", "ONCE",
     0, "PLACEBO", "2013-07-26", "2013-07-26", "ONCE",
     0, "PLACEBO", "2013-07-27", "2013-07-27", "ONCE",
     0, "PLACEBO", "2013-07-28", "2013-07-28", "ONCE",
     0, "PLACEBO", "2013-07-29", "2013-07-29", "ONCE",
     0, "PLACEBO", "2013-07-30", "2013-07-30", "ONCE",
     0, "PLACEBO", "2013-07-31", "2013-07-31", "ONCE",
     0, "PLACEBO", "2013-08-01", "2013-08-01", "ONCE"
   )
 )
```

What do you think about this?

**Comment by bundfussr** [comment]
> * The styler package (in my opinion) does a bad job with tribbles:
>   Screenshot 2023-05-22 at 13 19 15
> *

The styler package preserves the alignment if all columns are completely aligned. In the example the entries in the first row are not aligned with those in the following rows.

**Comment by millerg23** [comment]
> > The styler package preserves the alignment if all columns are completely aligned. In the example the entries in the first row are not aligned with those in the following rows.
> 
> @bundfussr Thanks for the quick reply. Unfortunately, this did not work: Screenshot 2023-05-22 at 14 30 30
> 
> Do you have any other suggestions?

@StefanThoma its a bit strange, but you need to left align the first column, the rest of the columns can be right aligned, as you have,  I think this should work.

**Comment by StefanThoma** [comment]
> @StefanThoma its a bit strange, but you need to left align the first column, the rest of the columns can be right aligned, as you have, I think this should work.

Thanks @millerg23, I tried this out. It looks like everything has to be left aligned. 
Will update soon.

**Review by zdz2101 — CHANGES_REQUESTED** [review]
All the examples look to be addressed with the properly formatted styler stuff, after an overview from the pkgdown website from this branch things look good with the correct styler formatting stuff too

Only one function/script calls upon `library(admiral.test)` now in its example and its `create_query_data()` @bundfussr I don't think there is a way to get around this one is there?

File: renv/4.1/.Rprofile [hunk 1]
```diff
@@ -0,0 +1 @@
+source("renv/activate.R")
```
[the remark this answers is not in the document]
Comment (StefanThoma) [reply renv/4.1/.Rprofile:1]: Agree!

**Comment by StefanThoma** [comment]
@zdz2101 not sure what is the problem with the code style. 
When I run it locally everything seems fine. 
The error shown on GH actions is not helpful either. 
Do you know what's going on?

**Comment by bundfussr** [comment]
> Only one function/script calls upon `library(admiral.test)` now in its example and its `create_query_data()` @bundfussr I don't think there is a way to get around this one is there?

No, we have to keep it.

**Comment by bundfussr** [comment]
> The error shown on GH actions is not helpful either.

Seems to be a general issue on github:
image

@cicdguy , any ideas? Should we just wait?

File: renv/4.1/.Rprofile (hunk 1 above)
[the remark this answers is not in the document]
Comment (StefanThoma) [reply renv/4.1/.Rprofile:1]: Sorry, my Git magic is failing me. How do I do that? 
Tried to revert the file to previous commit, and tried to update to current devel branch, but it shows either way.

[the remark this answers is not in the document]
Comment (zdz2101) [reply renv/4.1/.Rprofile:1]: fastest solution might be to just start a fresh branch again and copy over these changes from the `.R`/`.Rd` s, not the best git wizard myself

[the remark this answers is not in the document]
Comment (StefanThoma) [reply renv/4.1/.Rprofile:1]: Sad! =)
Alright, let me get this done!

**Pull request closed without merging**